source/usage-examples/struct-tagging.txt

```diff
@@ -0,0 +1,69 @@
+===========================
+Set Custom BSON Field Names
+===========================
+
+.. default-domain:: mongodb
+
+You can specify the way that the Go Driver converts Go
+structs to :manual:`BSON </reference/bson-types/>` using 
+**struct tags**.
+
+The following code declares a ``BlogPost`` struct, where the ``WordCount`` field
+has a struct tag that sets a custom BSON field name of ``word_count``.
+The driver marshals the other fields as the lowercase of the struct
+field name by default:
+
+.. literalinclude:: /includes/usage-examples/code-snippets/struct-tag.go
+   :start-after: begin struct
+   :end-before: end struct
+   :language: go
+   :dedent:
+
+The following example creates a ``BlogPost`` instance and inserts it
+into the ``posts`` collection of the ``sample_training`` database.
+During the insert operation, the driver interprets the custom field name
+struct tag to marshal the ``WordCount`` struct field as ``word_count``:
```

Comment:
```suggestion
During the insert operation, the driver interprets the ``WordCount`` struct, the 
custom field name struct tag, to marshal as ``word_count``:
```
Suggestion: I think its more clear and easier to read this way

Reply:
addressed